### PR TITLE
chore: Adapt to avs 7 to enable large audio calls

### PR DESCRIPTION
**Pull request by thisisamir98**

### CODE REVIEW

File: src/script/calling/Call.ts [hunk 1]
```diff
@@ -81,6 +90,35 @@ export class Call {
     return this.participants().find(({user, clientId}) => user.isMe && this.selfClientId === clientId);
   }
 
+  addAudio(audioId: string, stream: MediaStream) {
+    this.audios[audioId] = {audioElement: null, stream};
+  }
+
+  removeAudio(audioId: string) {
+    delete this.audios[audioId];
```
Comment (bennycode) [inline src/script/calling/Call.ts:99]: How about stopping the stream before removing it?

File: test/unit_tests/calling/CallingRepositorySpec.js [hunk 2]
```diff
@@ -68,19 +68,17 @@ describe('CallingRepository', () => {
...
+          audioOutput: ko.pureComputed(() => 'test'),
+        },
+      });
       activeCall.state(CALL_STATE.MEDIA_ESTAB);
       spyOn(callingRepository.callState, 'activeCalls').and.returnValue([activeCall]);
       spyOn(amplify, 'publish').and.returnValue(undefined);
       const conversationId = createRandomUuid();
       const conversationType = CONV_TYPE.ONEONONE;
-      const callType = CALL_TYPE.NORMAL;
+      const callType = 0;
```
Comment (bennycode) [inline test/unit_tests/calling/CallingRepositorySpec.js]: Why not using an enum here?

**Review by bennycode — CHANGES_REQUESTED** [review]

File: src/script/calling/Call.ts (hunk 1 above)
Comment (thisisamir98) [reply src/script/calling/Call.ts:99]: done in https://github.com/wireapp/wire-webapp/pull/10657/commits/487f5113d8bed815d1c9a5a51f749303f5ebf985

File: test/unit_tests/calling/CallingRepositorySpec.js (hunk 2 above)
Comment (thisisamir98) [reply test/unit_tests/calling/CallingRepositorySpec.js]: done in https://github.com/wireapp/wire-webapp/pull/10657/commits/74d3191599d112de83e13492cc0d84c1ea4db454

**Review by bennycode — APPROVED** [review]

**Pull request merged**